.travis.yml

```
@@ -46,6 +46,6 @@ install:
 
 script:
 # Run linter
-- if [ "$RUN_CS_FIXER" = "true" ]; then ./php-cs-fixer fix --diff; fi
+- if [ "$RUN_CS_FIXER" = "true" ]; then ./php-cs-fixer fix --dry-run --diff; fi
```

More concise option to consider:
```suggestion
- test "$RUN_CS_FIXER" == "true" && ./php-cs-fixer fix --dry-run --diff
```

---

I like it